src/ui/base/select.re

```diff
@@ -1,8 +1,9 @@
+open React.Dom.Dsl.Html;
 open Batteries;
 
 [@react.component]
 let make = (~options, ~compare=?, ~value, ~on_change) => {
-  let compare = Utils.fix_opt_arg(compare) |> Option.default((==));
+  let compare = Option.default((==), compare);
```

Comment:
Is comparing for reference equality the right thing to here ?

Reply:
In Reason == and != is used for structural (in-)equality and === and !== is used for reference (in-)equality.